apps/client/src/components/Location/LocationFinder.js

```diff
@@ -40,12 +44,19 @@ const LocationFinder = (props) => {
     // Pass the GraphQL error to the HOC
     setErrorMessage(graphqlError);
 
-    if (postalCode && houseNumberFull && !loading && (data || graphqlError)) {
+    if (
+      houseNumber &&
```

Comment:
This is a bit hard to read, maybe we should create a const with a readable name.

Reply:
How about this?